plugins/woocommerce-admin/client/customize-store/design-with-ai/actions.ts

```diff
@@ -210,6 +217,8 @@ const assignAPICallLoaderError = assign<
 	designWithAiStateMachineEvents
 >( {
 	apiCallLoader: () => {
+		recordEvent( 'customize_your_store_ai_wizard_error' );
```

Comment:
Is it possible to get any context for this error? I think it can be valuable if we can gather the reason if any.

Reply:
It's possible! ll create a follow-up task for it. We have other tracks that logs error message so I think it's okay for now. 🙂